### PR TITLE
ddl: skip write binlog when ddl type is update tiflash replica status

### PR DESCRIPTION
**Pull request by crazycs520**
Signed-off-by: crazycs520 <crazycs520@gmail.com>

### What problem does this PR solve?

As title said.

Related PR: https://github.com/pingcap/tidb-binlog/pull/941

### What is changed and how it works?

Update TiFlash replica is a TiDB internal DDL, It's used to update the table's TiFlash replica available status, and It should not write the binlog.

### Related changes

- Need to cherry-pick to the release branch

### Check List 

Tests 

- Manual test (add detailed scripts or steps below)

Side effects

### Release note 

- No.

### CODE REVIEW

**Comment by crazycs520** [comment]
@july2993  @GregoryIan @zimulala PTAL

File: ddl/ddl_worker.go [hunk 1]
```diff
@@ -511,12 +511,26 @@ func (w *worker) handleDDLJobQueue(d *ddlCtx) error {
 	}
 }
 
+func skipWriteBinlog(job *model.Job) bool {
+	switch job.Type {
+	// ActionUpdateTiFlashReplicaStatus is a TiDB internal DDL,
+	// it's used to update table's TiFlash replica available status.
+	case model.ActionUpdateTiFlashReplicaStatus:
```
Comment (GregoryIan) [inline ddl/ddl_worker.go:518]: also for ActionSetTiFlashReplica？

Comment (crazycs520) [reply ddl/ddl_worker.go:518]: `ActionSetTiFlashReplica` is not a TiDB internal DDL.

**Pull request merged**